elements/pfe-sass/extends/_extends.scss

```diff
@@ -57,45 +57,45 @@
   }
 }
 
-%ordered-list-primary {
-  // size of the font inside of the circle
-  --FontSize: var(--pfe-list-FontSize, 24px);
-  // size of the space between the circle and the content
-  --gap: var(--pfe-list--gap, 24px);
-  // amount of space between list items
-  --margin: var(--pfe-list--margin, 24px);
-  // font color of the numbers in the circle
-  --Color: var(--pfe-list--Color, rgba(238, 0, 0, 1));
-  // background color of circle
-  --BackgroundColor: var(--pfe-list--BackgroundColor, rgba(238, 0, 0, 0.07));
+// Mixins and extends can't use the LOCAL-VARIABLES map because we would be
+// at risk of overriding an elements map if they call our mixin. We need to
+// use just regular sass variables and pfe-local and specify the $fallbacks.
+@mixin ordered-list-primary {
+  // First we create our Sass variables that we will use for fallbacks
+  $gap: pfe-var(content-spacer);
   // size of the circle
-  --size: var(--pfe-list--size, 56px);
-  // calculated radius of the circle
-  --size-radius: calc(var(--size) / 2);
-  // need to set these explicitly to override default list styles
-  --Margin: var(--pfe-list--Margin, var(--margin));
-  --PaddingLeft: var(--pfe-list--PaddingLeft, 0);
+  $size: 56px;
+  // we need to wrap this in a calc function so it can be dynamic.
+  // rgb value of the color theme of the circle
+  $color: (238, 0, 0);
+  $font-size: $pf-global--FontSize--2xl;
+
+  // Then we create the same Sass variables as CSS variables so we can use
+  // them in CSS calc functions
+  --pfe-#{$LOCAL}__gap: #{$gap};
```

Comment:
`--pfe-list__gap` is not a valid variable name because it specifies a region but not property.  I suspect you wanted `--pfe-list--gap` instead to support gap styles on the overall component.